genome_designer/genome_finish/graph_contig_placement.py

```diff
@@ -145,6 +247,69 @@ def avg_coverage(bam, chromosome, start, end):
     return avg_coverage
 
 
+def add_me_alignment_to_graph(G, contig_alignment_bam):
```

Comment:
I'm going to trust this works. You'll need write up an explanation of how this works for the supplement / readthedocs.